master/getting-started/kubernetes/installation/hosted/calico.yaml

```diff
@@ -152,7 +152,9 @@ spec:
             # Set Felix endpoint to host default action to ACCEPT.
             - name: FELIX_DEFAULTENDPOINTTOHOSTACTION
               value: "ACCEPT"
-            # Configure the IP Pool from which Pod IPs will be chosen.
+            # The default IPv4 pool to create on startup if none exists. Pod IPs will be
+            # chosen from this range. Changing this value after installation will have
+            # no effect. This should fall within `--cluster-cidr`.
```

Comment:
@ozdanborne maybe we want a link to your new doc here for people who do want to change it?